R/armax.sim.R

```diff
@@ -3,6 +3,10 @@
 ## Copyright (c) Felix Andrews <felix@nfrac.org>
 ##
 
+
+#' @importFrom stats filter na.pass
+#'
+#' @export
 armax.sim <-
```

Comment:
I think it would make sense to add the annotations in armax.R here instead.

Reply:
Done!